src/tests/api-tests.ts

```diff
@@ -1745,8 +1745,30 @@ describe('api tests', () => {
     };
     await db.updateMempoolTxs({ mempoolTxs: [mempoolTx] });
 
+    const blockMetadata = {
```

Comment:
can you please write another test to include metadata instead of modifying the existing tests?